packages/web/pages/docs/rest-api.mdx

```diff
@@ -47,3 +47,99 @@ You then pass your API key in each request with one of these methods:
 
 - Include the API key in the request headers: `X-API-KEY=...`
 - Include the API key as a query string parameter: `?apiKey=...`
+
+## API endpoints
+
+All API endpoints expect the body to be a JSON string and the header Content-Type to be "application/json".
+
+### /api/sendMail
+
+Send an email using your configured sendmail.
+
+**Authentication required**
+
+Yes, using an [API key](#api-keys)
+
+**Request parameters**
+
+You can either use /api/sendMail to send raw html or a template.
+
+- [Standard nodemailer options](#standard-nodemailer-options)
```

Comment:
I think it would be better to reference https://www.mailing.run/docs/sending-email#calling-sendmail  than nodemailer docs. The nodemailer section at the end feels very random unless you get there from this link.